Content.Server/CartridgeLoader/Cartridges/CrewManifestCartridgeSystem.cs

```diff
@@ -0,0 +1,88 @@
+﻿using Content.Server.CrewManifest;
+using Content.Server.Station.Systems;
+using Content.Shared.CartridgeLoader;
+using Content.Shared.CartridgeLoader.Cartridges;
+using Content.Shared.CCVar;
+using Robust.Shared.Configuration;
+
+namespace Content.Server.CartridgeLoader.Cartridges;
+
+public sealed class CrewManifestCartridgeSystem : EntitySystem
+{
+
+    [Dependency] private readonly CartridgeLoaderSystem? _cartridgeLoader = default!;
+    [Dependency] private readonly StationSystem _stationSystem = default!;
+    [Dependency] private readonly CrewManifestSystem _crewManifest = default!;
+    [Dependency] private readonly IConfigurationManager _configManager = default!;
+
+    private const string CartridgePrototypeName = "CrewManifestCartridge";
+
+    private bool _allowed = true;
+
+    public override void Initialize()
+    {
```

Comment:
Needs commenting (_allowed could mean anything), also sort the dependencies please.

Reply:
That's true. Renamed variable and wrote a few lines. Thank you